config.go

```diff
@@ -338,3 +338,31 @@ func nodejs(c *Config) error {
 
 	return nil
 }
+
+// python config.
+func python(c *Config) {
+	if c.Proxy.Command == "" {
+		c.Proxy.Command = "python app.py"
+	}
+
+	// Only add build & clean hooks if a requiremnts.txt exists
+	if !util.Exists("requirements.txt") {
+		return
+	}
+
+	// Set PYTHONPATH env
+	if c.Environment == nil {
+		c.Environment = config.Environment{}
+	}
+	c.Environment["PYTHONPATH"] = ".pypath/"
+
+	// Copy libraries into .pypath/
+	if c.Hooks.Build == "" {
+		c.Hooks.Build = `mkdir -p .pypath/ && pip install -r requirements.txt -t .pypath/`
```

Comment:
@arikfr yea that'd be the case for native deps, have to resort to Docker at that point I guess, or CI.

@AaronO cool sounds good! We can tweak it later if necessary. It would be cool if there was maybe a small tool that could symlink the deps used, I think I added support for resolving links (forget now haha)